Makefile

```diff
@@ -43,6 +43,9 @@ lint-fmt:
 	@echo Checking python file formatting with the black code style checker
 	pipenv run black --line-length=100 --check $(dirs)
 
+lint-misc:
```

Comment:
Why not introducing `lint-mitre` and then introduce `lint-misc` whenever another "misc" linting is added in the future? And the `lint-misc` will be running the mitre lint and the newly introduced one.

Reply:
That makes sense! In both cases, the MITRE check will run during make lint, so customers won't need to make any changes to their workflows